source/validator/contracts/Validator.sol

```diff
@@ -505,8 +506,8 @@ contract Validator {
   }
 
   /**
-    * @notice Checks token has valid ERC721 interface
-    * @param tokenAddress address potential ERC721 token address
+    * @notice Checks token has valid ERC721 or ERC1155 interface
```

Comment:
instead of `ERC721 or ERC1155 ` can it just say `token has valid interface` ? as it works with all interfaces